MAUI/UIKit/maui-ui-designs.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title: MAUI UI Design | Syncfusion
+description: Essential UI Kit contains elegantly designed XAML templates for MAUI apps. These templates are compatible with Android, iOS, and UWP platforms.
+platform: maui
+control: MAUI UI Kit
+documentation: ug
+---
+
+# Syncfusion MAUI UI Designs 
+
+The Essential UI Kit includes 101 screens which is under the following categories. 
+
+## Forms	
+
+* [Simple Login Page](https://github.com/syncfusion/essential-ui-kit-for-.net-maui/blob/master/EssentialUIKit/Views/Forms/SimpleLoginPage.xaml)
```

Comment:
In the categories some screens are removed in MAUI check and remove

Reply:
Removed and Ensured in Latest Commits